client/influxdb.go

```diff
@@ -32,6 +33,18 @@ const (
 type Query struct {
 	Command  string
 	Database string
+
+	// Chunked tells the server to send back chunked responses. This places
+	// less load on the server by sending back chunks of the response rather
+	// than waiting for the entire response all at once.
+	Chunked bool
+
+	// ChunkSize sets the maximum number of rows that will be returned per
+	// chunk. Chunks are either divided based on their series or if they hit
+	// the chunk size limit.
+	//
+	// Chunked must be set to true for this option to be used.
+	ChunkSize int
```

Comment:
If you just send chunked, it will use the default. If you require the chunk size to be set, you can never use the default and have to set the value. I guess the question is if we care about that or we want the client to have a default value that it uses that is separate from the server's.